website/docs/how-to/how-to-set-up-group-sso-sync.md

```diff
@@ -0,0 +1,41 @@
+---
+title: How set up user group SSO syncing
+---
+
+:::info availability
+
+User group syncing is planned to be released in Unleash 4.18 and will be available for enterprise customers.
+
+:::
+
+This guide takes you through how to configure your [user groups](../user_guide/rbac#user-groups) to automatically populate users through Single Sign On (SSO). Refer to [setting up Keycloak for user group sync](../../advanced/sso-keycloak-group-sync) for an end to end example. Note that the steps below require you to be logged in as an admin user.
+
+### Step 1: Navigate to SSO configuration {#step-1}
+
+Navigate to the "Single sign-on" configuration page.
+
+![The Unleash Admin UI with the steps highlighted to navigate to the Single sign-on configuration.](/img/setup-sso-group-sync-1.png)
+
+### Step 2: Enable Group Syncing {#step-2}
+
+Turn on "Enable Group Syncing" and input a field a for "Group Field JSON Path". The latter is the JSON path in the token response where your group properties are located, this is up to your SSO provider, a full example for Keycloak can be [found here](../../advanced/sso-keycloak-group-sync). Once you're happy, save your configuration.
```

Comment:
Some small rewording suggestions for clarity here. This is a tricky paragraph. I think the behavior of the "Group Field JSON Path" should be explained in the reference docs (rbac).

```suggestion
Turn on "Enable Group Syncing" and enter a value a for "Group Field JSON Path". Refer to the [User group SSO integration documentation](../user_guide/rbac.md#user-group-sso-integration) for more information or to the [how-to guide for integrating with Keycloak](how-to-set-up-keycloak-sso.md) for a practical example. 

The value is the JSON path in the token response where your group properties are located, this is up to your SSO provider, a full example for Keycloak can be [found here](../../advanced/sso-keycloak-group-sync). Once you're happy, save your configuration.
```